chef_master/source/install_supermarket.rst

```diff
@@ -291,6 +291,57 @@ When the bootstrap operation is finished, do the following:
 
       $ sudo chef-client
 
+Install Supermarket Directly (without a cookbook)
+=====================================================
+While there are many benefits to using the cookbook method to install Supermarket, there are also cases where it's simpler to set up the Supermarket installation manually. These steps will walk you through the process of manually configuring your private Supermarket server.
+
+Before following these steps, be sure to complete the OAuth setup process detailed in the `Chef Identity </install_supermarket.html#chef-identity>`__ section of this guide.  
+
+#. `Download <https://downloads.chef.io/supermarket/>`__ the correct package for your operating system from ``downloads.chef.io``. 
+
+#. Install Supermarket using the appropriate package manager for your distribution:
+
+   * For Ubuntu:
+
+     .. code-block:: bash
+
+        dpkg -i /path/to/package/supermarket*.deb
+
+   * For RHEL / CentOS:
+
+     .. code-block:: bash
+
+        rpm -Uvh /path/to/package/supermarket*.rpm
+
+#. Run the ``reconfigure`` command to complete the initial installation:
+
+   .. code-block:: none
+
+      sudo supermarket-ctl reconfigure
```

Comment:
Why this initial reconfigure before creating `/etc/supermarket/supermarket.json`?

Reply:
This initial reconfigure creates the `/etc/supermarket` directory

Reply:
OK.